en/docs/how-tos/adding-apis/importing-a-group-of-apis-from-wso2-api-manager.md

```diff
@@ -37,6 +37,18 @@ Follow the instructions below to generate a Microgateway project for a group of
 
 2.  Import the APIs, which are published in WSO2 API Manager, that correspond to a specific Microgateway label.
 
+    !!! note
```

Comment:
```suggestion
    !!! note - Importing APIs from APIM 4.x versions
```

Reply:
It's not possible to add a header to a note. Therefore, added this as a bold text inside the note from https://github.com/wso2/docs-mg/pull/184/commits/e859e260fbfe0c94a1b89bed7dab4606bc37b31c